lib/liquid/expression.rb

```diff
@@ -29,5 +31,18 @@ def self.parse(markup)
       end
     end
 
+    def self.format(value)
+      if INVERTED_LITERALS.key?(value)
+        INVERTED_LITERALS[value].dup
```

Comment:
In my head the contract of `format` was to return a modifiable string, so that other places can `<<` to the result without worrying about modifying internal state.